### ISSUE TITLE
Mappingservice: Can't map to a mref or categorical mref

### ISSUE THREAD

**Issue by marijevdgeest** [molgenis/molgenis#4913]
#### Reproduce
[model.xlsx](https://github.com/molgenis/molgenis/files/289655/model.xlsx)
[source.xlsx](https://github.com/molgenis/molgenis/files/289683/source.xlsx)

- Upload **model.xlsx** and **source.xlsx**
- Start new mapping project, select **rdconnect_Sample** as the target entity.
- Add **source** as the source entity

**Mapping categorical mref:**

- Open the **Diagnosis Type** attribute in mapping project. 
- Add algorithm: 

```
var diagnosis = []
diagnosis.push($('Diagnosistype').value().toUpperCase());
diagnosis;
```
or hardcoded:
`['CLINICAL']`

**Mapping mref**

- Open **Disease** attribute in mapping project.
- add algorithm:

```
var result = [];
var code = $('ORPHA').value()
var prefix = 'urn:miriam:orphanet:Orphanet_'
if (code) {
	result.push(prefix + code)
} 
result;
```

#### Expected
**Mapping categorical mref:**
Result shown in preview window.

**Mapping mref:**
Result shown in preview window, no warnings in log.

#### Observed
**Mapping categorical mref:**
Result not shown in preview window, instead it shows 'invalid script' and a never-ending spinner.
Array can not be converted to html:

> For "?html" left-hand operand: Expected a string or something automatically convertible to string (number, date or boolean), but this has evaluated to a sequence (ArrayList wrapped into f.t.DefaultListAdapter)

http://freemarker.org/docs/ref_builtins_sequence.html

**Mapping mref:**
No result shown due to ftl error (as described with the categorical mrefs).
Logs shows warning:

> WARN  o.m.data.mysql.MysqlRepository - Lookup: Couldn't find rdconnect_Disease for id urn:miriam:orphanet:Orphanet_1390

Although the urn is found in the index and database.

When adding the urn hardcoded:
`['urn:miriam:orphanet:Orphanet_1390']`
Only the ftl error appears.

**Comment by marijevdgeest** [comment]
> query

**Comment by Mark-de-Haan** [comment]
> Start new mapping project, select rdconnect_Sample as the target entity.
There is no entity called rdconnect_Sample ;) only Disease and DiagnosisType

**Comment by Mark-de-Haan** [comment]
Mapping of categorical_mref fixed in #4941
Mapping of mref bug is caused by #4733. A temporary fix would be to not use the IRI as an identifier!
